### ISSUE TITLE
Homey (1.5.12) doesn't recognize Fibaro Motion Sensor v3.3 

### ISSUE THREAD

**Issue by iZwag** [athombv/homey#2905]
**Metadata**

* **Homey Version**: 1.5.12
* **Homey Log Code**: NA
* **Smartphone Platform**: NA, using Windows Desktop App 
* **Smartphone App Version**: NA

**If related to a Homey App:** 
* **Homey App Version**: Fibaro 1.5.18
* **Homey App Log Code**: NA 
* **Affected Devices (preferably product codes)**: Fibaro Motion Sensor FGMS-001 ZW5 3.3

**What did you try to achieve?**

Tried to add the Fibaro Motion Sensor FGMS-001 ZW5 3.3 to the Homey. 

**What did you expect as result?**

Expected it to get added just like all other Z-Wave devices from Fibaro, which work perfectly. I have several Fibaro Motion Sensor FGMS-001 ZW5 v3.2 that also have no problems being added.

**What was the result?**

After adding the device to a zone via the Fibaro-tab in add devices, the normal adding procedures executes, but at the final step, this error occurs (see attached image):
![error](https://user-images.githubusercontent.com/6748165/47777249-9da88c80-dcf4-11e8-8194-9b55ca715f7e.PNG)

I have tried this on three independent Homeys, all have the same version of Homey (1.5.12) and Fibaro (1.5.18), and all have problems with the same version of the Motion Sensor. I have tried 3 different Fibaro Motion Sensors v3.3, and they all have the same issue. I have no problems adding Fibaro Motion Sensor v3.2 to all the affected Homeys.

**Any other remarks**
Adding images of the node information for the Fibaro Motion Sensor v3.2 (working) and Fibaro Motion Sensor v3.3 (not working) for reference here:
![3-2-info](https://user-images.githubusercontent.com/6748165/47777577-540c7180-dcf5-11e8-9840-146592b26746.PNG)
![3-3-info](https://user-images.githubusercontent.com/6748165/47777578-540c7180-dcf5-11e8-9b92-1aec0c3e9723.PNG)

Seems to me like the current Homey version doesn't support the new firmware of the Fibaro Motion Sensor. Can you please do something about this ASAP?
I have tried every combination of factory resetting the sensors, adding/removing them, physical distance to the Homey when adding, resetting the Z-Wave network entirely. I have also tried the beta-/unstable version of Fibaro without luck.

- Jon / iZwag

**Comment by Divemasters** [comment]
Dear,

I have the same sensor and the same problem...

**Comment by melvinwhb** [comment]
I have the same problem. Any way to downgrade the Motion sensor to V3.2 for the time being?

**Comment by EspenWathne** [comment]
If you look at the product ID v 3.3 has id 4098 while v 3.2 has ID 4097. If I am not mistaken this productid is not defined in your version of the Fibaro app. Try to upgrade the Fibaro app to the latest beta (v2.1.0) and see if that will fix the issue.
Disclaimer: Beta is not production quality. :)

**Comment by melvinwhb** [comment]
That is correct, but neither is it in Beta V2.1.0 apparently. I am running the latest beta and experiencing the same problem unfortunately.

**Comment by iZwag** [comment]
> If you look at the product ID v 3.3 has id 4098 while v 3.2 has ID 4097. If I am not mistaken this productid is not defined in your version of the Fibaro app. Try to upgrade the Fibaro app to the latest beta (v2.1.0) and see if that will fix the issue.
> Disclaimer: Beta is not production quality. :)

Thank you for suggesting that, but as I said in the last sentence before ending my original post (however without explicitly listing the beta _version_ I was using), I have already tried the latest beta version. It also does not fix the issue.

Do you know any way of either manually adding the product-ID to a list of recognized Fibaro devices, or a way of downgrading the Fibaro sensor?

**Comment by iZwag** [comment]
> I have the same problem. Any way to downgrade the Motion sensor to V3.2 for the time being?

Not sure if this is much help, but at least I can confirm that even buying a Fibaro Home Center Lite does not grant you the ability to downgrade the firmware / product-ID of Motion Sensor. I read a post suggesting this somewhere, so I tried it out, with no luck. My Fibaro Motion Sensors v3.3 are still useless at this point.
